### PR TITLE
[SKILLONOMY-104] Add course status

### PR DESCRIPTION
**Pull request by dyudyunov**
Changes:
  - Added price to course advanced settings (cms);
  - Added translations for statuses;
  - For anabling visibility for courses with enrollment start in future
need to change FEATURE setting: 'DISABLE_START_DATES': True.

Related PR to theme: https://github.com/raccoongang/edx-theme/pull/1772

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-104

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: common/lib/xmodule/xmodule/course_module.py [hunk 1]
```diff
@@ -168,14 +168,20 @@ def to_json(self, values):
...
     edeos_secret = String(display_name='EdEos Secret', scope=Scope.settings, default="",
-                               help=_('EdEos Secret'))
+                          help=_('EdEos Secret'))
     edeos_key = String(display_name='EdEos Key', scope=Scope.settings, default="",
-                            help=_('EdEos Key, also known as Client Id'))
+                       help=_('EdEos Key, also known as Client Id'))
+    price = String(
+        display_name=_("Course Price"),
+        help=_("Set the course price."),
+        default="Free",
```
Comment (OPersian) [inline common/lib/xmodule/xmodule/course_module.py]: Maybe this should be i10n-ed too, but I'm not sure

File: lms/static/js/discovery/views/course_card.js [hunk 2]
```diff
@@ -33,10 +33,17 @@
                 var data = _.clone(this.model.attributes);
                 var userLanguage = '',
                     userTimezone = '';
+                var currentDate = new Date();
                 if (this.model.userPreferences !== undefined) {
                     userLanguage = this.model.userPreferences.userLanguage;
                     userTimezone = this.model.userPreferences.userTimezone;
                 }
+                var afterEnrollStarted = (currentDate < new Date(data.start)) ? "The enrollment is open" : "The course is on";
```
Comment (OPersian) [inline lms/static/js/discovery/views/course_card.js]: Are we sure that enrollment **always** closes once a course starts?

Comment (OPersian) [reply lms/static/js/discovery/views/course_card.js]: Also, the var's naming `afterEnrollStarted` and its value seems a bit unclear to me...

Is there any way to separate start / enroll logic?

File: common/lib/xmodule/xmodule/course_module.py (hunk 1 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/course_module.py]: I don't think it's a good idea to translate default status depending on course creators' language. In this case courses from different creators will be mixed in LMS (free vs бесплатно for example) if they will leave it as default.

File: lms/static/js/discovery/views/course_card.js (hunk 2 above)
Comment (dyudyunov) [reply lms/static/js/discovery/views/course_card.js]: Reworked, Thanks!

File: lms/static/js/discovery/views/course_card.js [hunk 3]
```diff
@@ -33,10 +33,23 @@
                 var data = _.clone(this.model.attributes);
                 var userLanguage = '',
                     userTimezone = '';
+                var currentDate = new Date();
+                var courseStatus;
```
Comment (GlugovGrGlib) [inline lms/static/js/discovery/views/course_card.js]: Please initialize `courseStatus` variable with the string "The course is on" and then remove `else` statement at line 47
```suggestion
                var courseStatus = "The course is on";
```

Comment (dyudyunov) [reply lms/static/js/discovery/views/course_card.js]: Done, thanks.

File: lms/static/js/discovery/views/course_card.js (hunk 2 above)
Comment (OPersian) [reply lms/static/js/discovery/views/course_card.js]: Approved, although I'm not sure I understand the business logic well (still confused by enrolling start date VS course start date logic).

**Pull request merged**